code/modules/mob/living/say.dm

```diff
@@ -138,21 +138,26 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 					var/mob/M = I
 					listening += M
 					hearturfs += M.locs[1]
-					for(var/obj/O in M.contents)
-						var/obj/item/clothing/worn_item = O
-						if((O.flags_atom & USES_HEARING) || ((istype(worn_item) && worn_item.accessories)))
-							listening_obj |= O
+					for(var/obj/hearing_obj in M.contents)
+						var/obj/item/clothing/worn_item = hearing_obj
+						if((hearing_obj.flags_atom & USES_HEARING) || ((istype(worn_item) && worn_item.accessories)))
+							listening_obj |= hearing_obj
+							for(var/obj/item/device/radio/listening_bug/bug in hearing_obj.contents)
+								listening_obj |= bug
```

Comment:
i dont like this typecheck for hearing within contents, snowflakey typechecks are ... bad. given it's iterating over all the hearing_obj.contents anyway, you might as well just make it a trait (or just use the existing USES_HEARING)